### PR TITLE
feat(Dataviz): Date & Time ranges

### PR DESCRIPTION
**Pull request by gjou-tlnd**
**What is the problem this PR is trying to solve?**

TDS needs time & datetime ranges
image

image

**What is the chosen solution to this problem?**

**Please check if the PR fulfills these requirements**

* [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/dataviz/src/components/BoxPlot/BoxPlot.component.test.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ describe('Boxplot chart', () => {
 					q1: 8,
 					q2: 90,
 					median: 58,
-					mean: 59.79
+					mean: 59.79,
```
Comment (romainseb) [inline packages/dataviz/src/components/BoxPlot/BoxPlot.component.test.tsx:15]: Is it intended ?

**Review by romainseb — APPROVED** [review]

File: packages/dataviz/src/components/BoxPlot/BoxPlot.component.test.tsx (hunk 1 above)
Comment (gjou-tlnd) [reply packages/dataviz/src/components/BoxPlot/BoxPlot.component.test.tsx:15]: I applied prettier on the whole src directory, do you mean the prettier config is not correct?

Comment (ncomont) [reply packages/dataviz/src/components/BoxPlot/BoxPlot.component.test.tsx:15]: Yes, it is correct. The goal is to facilitate reading of the diff when an attribute is added.

**Comment by ncomont** [comment]
I can't manage to change the value from the inputs
https://user-images.githubusercontent.com/26482371/104733301-c59e3200-573e-11eb-96a7-2630102cdd23.mp4

**Review by ncomont — APPROVED** [review]

**Pull request merged**